packages/design/src/Form/components/GenderId/GenderId.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import { type Meta, type StoryObj } from '@storybook/react';
+import { GenderIdPattern } from './GenderId.js';
+
+const meta: Meta<typeof GenderIdPattern> = {
+  title: 'patterns/GenderIdPattern',
+  component: GenderIdPattern,
+  decorators: [
+    (Story, args) => {
+      const FormDecorator = () => {
+        const formMethods = useForm();
+        return (
+          <FormProvider {...formMethods}>
+            <Story {...args} />
+          </FormProvider>
+        );
+      };
+      return <FormDecorator />;
+    },
+  ],
+  tags: ['autodocs'],
+};
+
+export default meta;
+
+export const Default: StoryObj<typeof GenderIdPattern> = {
+  args: {
```

Comment:
Since the args for the component stories are mostly the same, it would be good to create a default config and use the spread operator to modify cases where it needs a different prop or two.

Reply:
Good point! Will update now!

Reply:
Updated!